Wpf/Microsoft.DataTransfer.WpfHost/Model/Statistics/ObservableErrorsTransferStatisticsFactory.cs

```diff
@@ -23,5 +24,10 @@ public ObservableErrorsTransferStatisticsFactory(ITransferStatisticsFactory defa
                 ? new ObservableErrorsTransferStatistics(defaultStatistics, errorDetailsProvider, SynchronizationContext.Current)
                 : defaultStatistics;
         }
+
```

Comment:
documentations for the public fields and functions missing.

Reply:
actually, these new properties along with the others are described in : Core/Microsoft.DataTransfer.ServiceModel/ConfigurationResources.resx

This is the WPF project. I noticed that there is no previous documentation  of other properties. Adding these params here does not make a difference, as the Azure Table adapter (even before my changes) is not supported in the UI/WPF.

Reply:
More than these, for example, AzureTableSourceAdapter, looks like we didn't follow at least doc the "public" functions? I am fine if the resx has the documentation for this file.

Reply:
resolved after discussing, that the current change follows the project documentation pattern.